app/modules/ads/module.js

```diff
@@ -101,62 +103,76 @@ class Ads {
+          }
+        },
+      });
+    }
+  }
 
+  callCheshireCat() {
     function getNextIncontentId(predictions) {
       return `incontent_boxad_${Object.keys(predictions).length + 2}`;
     }
 
-    if (context.get('bidders.prebid.bidsRefreshing.enabled')) {
-      this.cheshirecatPredictions = {};
+    const { context, slotService } = window.Wikia.adEngine;
+    const { billTheLizard } = window.Wikia.adServices;
 
-      context.set('bidders.prebid.bidsRefreshing.bidsBackHandler', () => {
-        const config = instantGlobals.wgAdDriverBillTheLizardConfig || {};
-        const bidderPrices = targeting.getBiddersPrices('mobile_in_content');
-
-        context.set('services.billTheLizard.projects', config.projects);
-        context.set('services.billTheLizard.timeout', config.timeout || 0);
-        context.set('services.billTheLizard.parameters.cheshirecat', {
-          bids: [
-            bidderPrices.bidder_1 || 0,
-            bidderPrices.bidder_2 || 0,
-            0,
-            bidderPrices.bidder_4 || 0,
-            0,
-            bidderPrices.bidder_6 || 0,
-            bidderPrices.bidder_7 || 0,
-            0,
-            bidderPrices.bidder_9 || 0,
-            bidderPrices.bidder_10 || 0,
-            bidderPrices.bidder_11 || 0,
-            bidderPrices.bidder_12 || 0,
-            bidderPrices.bidder_13 || 0,
-            bidderPrices.bidder_14 || 0,
-            bidderPrices.bidder_15 || 0,
-            bidderPrices.bidder_16 || 0,
-          ].join(';'),
-        });
+    const bidderPrices = targeting.getBiddersPrices('mobile_in_content');
+
+    context.set('services.billTheLizard.projects', this.cheshirecatConfig.projects);
+    context.set('services.billTheLizard.timeout', this.cheshirecatConfig.timeout || 0);
+    context.set('services.billTheLizard.parameters.cheshirecat', {
+      bids: [
+        bidderPrices.bidder_1 || 0,
+        bidderPrices.bidder_2 || 0,
+        0,
+        bidderPrices.bidder_4 || 0,
+        0,
+        bidderPrices.bidder_6 || 0,
+        bidderPrices.bidder_7 || 0,
+        0,
+        bidderPrices.bidder_9 || 0,
+        bidderPrices.bidder_10 || 0,
+        bidderPrices.bidder_11 || 0,
+        bidderPrices.bidder_12 || 0,
+        bidderPrices.bidder_13 || 0,
+        bidderPrices.bidder_14 || 0,
+        bidderPrices.bidder_15 || 0,
+        bidderPrices.bidder_16 || 0,
+      ].join(';'),
+    });
 
-        billTheLizard.projectsHandler.enable('cheshirecat');
-        billTheLizard.executor.register('catlapseIncontentBoxad', () => {
-          const slot = getNextIncontentId(this.cheshirecatPredictions);
+    this.cheshirecatCalled = true;
 
-          if (slot) {
-            slotService.disable(slot, 'catlapsed');
-          }
-        });
+    billTheLizard.projectsHandler.enable('cheshirecat');
+    billTheLizard.executor.register('catlapseIncontentBoxad', () => {
+      const slot = getNextIncontentId(this.cheshirecatPredictions);
+
+      if (slot) {
```

Comment:
Is it possible to be false?

Reply:
You never know with those sneaky string variables!!... but yes, it's not possible. Removed :x